### PR TITLE
Katello 2.3 Release Notes

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: docs/release_notes/release_notes.md [hunk 1]
```diff
@@ -14,7 +14,133 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
...
+
+### Docker
+ * Docker Content now hidden for composite cv ([#9610](http://projects.theforeman.org/issues/9610), [68312e2](http://github.com/katello/katello/commit/68312e2))
+ * Fix updating of docker upstream name ([#9423](http://projects.theforeman.org/issues/9423), [ab345f5](http://github.com/katello/katello/commit/ab345f5))
+ * Docker Content now hidden for composite cv ([#9610](http://projects.theforeman.org/issues/9610), [348c873](http://github.com/katello/katello/commit/348c873))
+ * Remove unused erroneous file ([#9539](http://projects.theforeman.org/issues/9539), [1c977c9](http://github.com/katello/katello/commit/1c977c9))
+
+ * fixing db/seeds.rb for katello, BZ 868910 ([#3272](http://projects.theforeman.org/issues/3272), [346f187](http://github.com/katello/katello/commit/346f187))
 
 ## Contributors
```
Comment (ehelms) [inline docs/release_notes/release_notes.md]: This list looks like all contributors for all time, whereas in the past we were only generating the contributors list for that release to include in the release notes.

Comment (stbenjam) [inline docs/release_notes/release_notes.md]: Ah OK, I'll come back to this. I want to add other repos too to the generate script, so we're making sure we get contributors to the web site + puppet modules.

**Comment by ehelms** [comment]
How about we just leave off the contributors list to start with instead of
removing people later?

**Comment by stbenjam** [comment]
@ehelms It is removed

**Comment by ehelms** [comment]
My bad, I misread what you meant. ACK

**Comment by stbenjam** [comment]
D'oh, wrong branch.  @ehelms Reverting this: https://github.com/Katello/katello.org/pull/177

New PR against KATELLO-2.3: https://github.com/Katello/katello.org/pull/176

**Pull request merged**